README.md

```diff
@@ -17,41 +17,53 @@
 </p>
 
 
-Transformers Interpret is a model explainability tool designed to work exclusively with 🤗 transformers.
+Transformers Interpret is a model explainability tool designed to work exclusively with the [transformers][transformers] package.
 
-In line with the philosophy of the transformers package tranformers interpret allows any transformers model to be explained in just two lines. It even supports visualizations in both notebooks and as savable html files.
+In line with the philosophy of the transformers package Tranformers Interpret allows any transformers model to be explained in just two lines. It even supports visualizations in both notebooks and as savable html files.
 
-This package stands on the shoulders of the the incredible work being done by the teams at [Pytorch Captum](https://captum.ai/) and [Hugging Face](https://huggingface.co/) and would not exist if not for the amazing job they are both doing in the fields of nlp and model interpretability respectively.
 
 ## Install
 
-I recommend Python 3.6 or higher, Pytorch 1.5.0 or higher, transformers v3.0.0 or higher, and captum 0.3.1 (**required**). The package does not work with Python 2.7.
-
-```bash
+```posh
 pip install transformers-interpret
 ```
 
+Required:
+
+* Python 3.6 or higher
+* Pytorch 1.5.0 or higher
+* [transformers][transformers] v3.0.0 or higher
+* captum 0.3.1 
+
+The package does not work with Python 2.7 or below.
+
+
+
+# Documentation
+
 ## Quick Start
 
-Let's start by importing the auto model and tokenizer class from transformers and initializing a model and tokenizer.
+Let's start by initializing a transformers' model and tokenizer, and running it through the `SequenceClassificationExplainer`.
 
-For this example we are using `distilbert-base-uncased-finetuned-sst-2-english` a distilbert model finetuned on a sentiment analysis task.
+For this example we are using `distilbert-base-uncased-finetuned-sst-2-english`, a distilbert model fine-tuned on a sentiment analysis task.
 
 ```python
 from transformers import AutoModelForSequenceClassification, AutoTokenizer
-model = AutoModelForSequenceClassification.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english")
-tokenizer = AutoTokenizer.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english")
-```
+model_name = "distilbert-base-uncased-finetuned-sst-2-english"
+model = AutoModelForSequenceClassification.from_pretrained(model_name)
+tokenizer = AutoTokenizer.from_pretrained(model_name)
 
-With both the model and tokenizer initialized we are now able to get get some explanations on some input text.
+# With both the model and tokenizer initialized we are now able to get explanations on an example text.
 
-```python
 from transformers_interpret import SequenceClassificationExplainer
-cls_explainer = SequenceClassificationExplainer("I love you, I like you", model, tokenizer)
+cls_explainer = SequenceClassificationExplainer(
+    "I love you, I like you", 
+    model, 
+    tokenizer)
```

Comment:
This broken invocation is clearer to read IMO - at first it took me a while where 'model' and 'tokenizer' were being used.